mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -526,9 +600,127 @@ def generate_to_meta(self, database: str, table: str) -> Metadata:
         if ff:
             meta.file_format = ff.lower()
 
+        # update metadata dict with glue table properties from table parameters
+        metadata_dict = _get_table_parameters(
+            metadata=meta,
+            table_parameters_resp=resp["Table"]["Parameters"],
+            glue_table_properties=glue_table_properties,
+            update_primary_key=update_primary_key,
+        )
+
+        meta = Metadata.from_dict(metadata_dict)
+
         return meta
 
 
+def _update_table_parameters(
+    metadata: Metadata,
+    table_input: dict,
+) -> dict:
+    """Adds the key, value pairs for primary_key and glue_table_properties from the
+    metadata to the parameters parameter in the table input. Will not add any key, value
+    pairs for keys in glue_table_properties that are protected properties defined in
+    _glue_table_properties_aws and/or other protected glue table properties such as
+    "primary_key".
+
+    Args:
+        metadata (Metadata): Metadata object.
+        table_input (dict): A dictionary defining the table metadata.
+
+    Returns:
+        dict: Updated table input with key, value pairs for primary_key and
+        glue_table_properties.
+    """
+    table_parameters = table_input["TableInput"]["Parameters"]
+
+    if metadata.primary_key:
+        table_parameters["primary_key"] = str(metadata.primary_key)
+
+    table_properties_protected = _glue_table_properties_aws + ["primary_key"]
+
+    for key, value in metadata._data.get("glue_table_properties", {}).items():
+        if key in table_properties_protected:
+            warnings.warn(
```

Comment:
would it be possible to include the table-name in this warning. I can image a scenario where you are iterating through 100's of tables and it'd be nice for the warning to be a bit more specific to pinpoint which tables are potentially problematic.

Reply:
Good point, I've added the table name in the warning now too.